assets/html/themeswap.js

```diff
@@ -1,10 +1,20 @@
 // Small function to quickly swap out themes. Gets put into the <head> tag..
 function set_theme_from_local_storage() {
-  // Browser does not support Web Storage, bail early.
-  if(typeof(window.localStorage) === "undefined") return;
-  // Get the user-picked theme from localStorage. May be `null`, which means the default
-  // theme.
-  var theme =  window.localStorage.getItem("documenter-theme");
+  // Intialize the theme to null, which means default
+  var theme = null;
+  // If the browser supports the localstorage and is not disabled then try to get the
+  // documenter theme
+  if(typeof(window.localStorage) != null) {
```

Comment:
I'm pretty sure we need to check the value, not the type here (`typeof(null) == "object"`):

```suggestion
  if(window.localStorage != null) {
```